### ISSUE TITLE
need Windows on arm version

### ISSUE THREAD

**Issue by pgs666** [balena-io/etcher#4310]
- **Etcher version:** 
- **Operating system and architecture:** 
- **Image flashed:** 
- **What do you think should have happened:** 
- **What happened:**I am using an arm Windows device，if can use an arm64 or arm64ec version I'll be very happy
- **Do you see any meaningful error information in the DevTools?**

**Comment by microhobby** [comment]
Yeah, the x86_64 version runs on Windows arm, but is slow as f*&$.
